src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -39,7 +40,9 @@ public class PersonCard extends UiPart<Region> {
     @FXML
     private Label remark;
     @FXML
-    private FlowPane assignedTier;
+    private FlowPane assignedStatus;
```

Comment:
Is there a need for both assignedStatus and assignedStatusAndTier ?

Reply:
Thank you for spotting this, I have amended it accordingly.